cmd/sqlcmd/sqlcmd.go

```diff
@@ -189,52 +189,52 @@ func SetScreenWidthFlag(args *SQLCmdArguments, rootCmd *cobra.Command) {
 }
 
 func setFlags(rootCmd *cobra.Command, args *SQLCmdArguments) {
```

Comment:
>tFlags(rootCmd [](http://example.com/codeflow?start=7&length=14)

you could just ignore this file and have Jyotika update it in her PR for cobra conversion

Reply:
Looks like that PR is already merged. Will check with her if there is any follow up PR that can include this change or else will take care of it as part of this PR.